test/unit/routes/signed-out-routes.test.js

```diff
@@ -0,0 +1,39 @@
+import { vi, beforeEach, describe, test, expect } from 'vitest'
+import { signedOut } from '../../../src/routes/signed-out-routes'
+
+import { signedOutPresenter } from '../../../src/presenters/signed-out-presenter'
```

Comment:
We shouldn’t need to stub a presenter in a route unit test, since it’s synchronous and simply returns hard-coded data without any transformation.